plotted a top N words per year plot as a barplot

Selkubi opened the pull request:
I created a plot of top N word per year as a stacked bar plot.

Selkubi commented on R/app_server.R:
```diff
@@ -153,6 +153,42 @@ app_server <- function(session,input, output) {
+     ggplot2::ggplot(words_per_year) +
+       ggplot2::aes(x = word, y = n, fill = as.factor(year)) +
+       ggplot2::geom_bar(position = "stack", stat = "identity") +
+       colorspace::scale_fill_discrete_sequential(palette = c("BluGrn"), name = "Year") +
```
I had difficulty to select a color palette that would work with 15 discrete colors. Recommendations to tackle this with ggplot2?

alwil commented:
As discussed, let's adapt the evolution of the words. You can continue pushing to this branch so the changes will be visible in the same pull request :)

Selkubi commented:
I have added the top N words plot with a numericInput option to select the number of words to plot. 
I also added a plot to show the evolution of the shown words. When a word is selected, it is highlighted.

cforgaci commented:
Sounds good. I'll have a look in a couple of weeks.(

cforgaci commented on R/app_ui.R:
```diff
@@ -88,14 +88,23 @@ app_ui <- function() {
+                                                                                   label = "Number of words:",
+                                                                                   value = 5,
+                                                                                   min = 0,
+                                                                                   max = 20                                                                )),
```
It is still possible to add more than 20 words. I don't see a way to limit that from the `numericInput`. If there's no other way that this can be done quickly, I suggest we switch to a slider input ranging from 5 to 20.

cforgaci reviewed (commented):
Thanks @Selkubi! I tested this and looks great! There's one small thing I noticed about the `numericInput` setting the number of top words. Let me know if it can be easily fixed.
